platform/osgi/platform-osgi-configadmin/src/main/java/org/codice/felix/cm/file/CopiedEncryptionService.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.felix.cm.file;
+
+import java.io.File;
+
+import org.apache.log4j.Logger;
+import org.keyczar.Crypter;
+import org.keyczar.KeyczarTool;
+
+public class CopiedEncryptionService {
```

Comment:
We would need the encryption service as a jar in order for this to work. We went with this strategy because we wanted the quick and easy solution since it's temporary and, by nature, hacky. We could possible package the EncryptionService as a jar in addition to it being a service but then we open the door to people overusing the jar instead of the service which is bad.

Reply:
If the encryption service is to be depended on by this bundle, it too needs to be a boot bundle and now we're really mucking with boot order needlessly for this PR. Additional research should be done to establish a proven system that handles low level bundles for us before we cross that bridge. 

With additional comments explaining the duel packaging, I'm sure a jar will be fine.

Reply:
Embedded encryption service impl 👍